File: BT.Python/Buổi 2 Python/1.4.1.py
def sum_1_to_n(n):
    s = 0
    for i in range(1, n+1):
        s += i
    return s

if __name__ == "__main__":
    n = int(input("nhap n: "))
    result = sum_1_to_n(n)
    print("tong 1..n = ", result)

File: BT.Julia/Buổi 2 Julia/1.4.1.jl
function sum_1_to_n(n)
    s = 0
    for i in 1:n
        s += i
    end
    return s
end
println("nhap n: ")
n = parse(Int, readline())
result = sum_1_to_n(n)
println(" tong 1..n = ", result)